laws/src/main/scala/cats/laws/discipline/ComonadTests.scala

```diff
@@ -26,10 +26,10 @@ trait ComonadTests[F[_], A, B] extends Laws {
   implicit def ArbFA: Arbitrary[F[A]] = ArbF.synthesize[A](ArbA)
   implicit def ArbFB: Arbitrary[F[B]] = ArbF.synthesize[B](ArbB)
 
-  def coflatmap[C: Arbitrary](implicit F: CoFlatMap[F], FC: Eq[F[C]]) = {
-    val laws = CoFlatMapLaws[F]
+  def coflatmap[C: Arbitrary](implicit F: CoflatMap[F], FC: Eq[F[C]]) = {
+    val laws = CoflatMapLaws[F]
     new ComonadProperties(
-      name = "coflatmap",
+      name = "coflatMap",
```

Comment:
I was unsure whether the convention on property names was lowercase or lowerCamelCase.